introduction.md

```diff
@@ -2,3 +2,11 @@ Here I am... Introducing myself to the dojo.. Yo!
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?